satellite/satellitedb/repairqueue_test.go

```diff
@@ -48,3 +53,78 @@ func TestRepairQueue(t *testing.T) {
 		}
 	})
 }
+
+func TestRepairQueueOrder(t *testing.T) {
```

Comment:
I think that the description of the test that you wrote in the issue could be added here.